README.md

```
@@ -1,123 +1,73 @@
+./start_dev.sh
 ```
 
-### Frontend:
-
-```bash
-cd frontend
-pnpm install
-pnpm run dev
+### Environment Variables (.env):
+Create `backend/.env`:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)